pymemcache/client/base.py

```diff
@@ -893,7 +893,8 @@ def _fetch_cmd(self, name, keys, expect_cas):
         remapped_keys = dict(zip(prefixed_keys, keys))
 
         # It is important for all keys to be listed in their original order.
-        cmd = name + b' ' + b' '.join(prefixed_keys) + b'\r\n'
+        # For Memcached Protocol correctness, if the list of prefixed_keys is empty, the name of the command will be immediately followed by the end of line.
+        cmd = name + b' ' + b' '.join(prefixed_keys) + b'\r\n' if prefixed_keys else name + b'\r\n'
```

Comment:
This is pretty complex. How about:

```py
cmd = name
if prefixed_keys:
    cmd += b' ' + b' '.join(prefixed_keys)
cmd += b'\r\n'
```

Then I also don't think you need a comment like about "protocol correctness" because the condition is obvious in the code.

Reply:
Seems simpler, sounds good thanks!